### DISCUSSION TITLE
Intergrating text element with the standard Joomla editor.

### DISCUSSION THREAD

**Discussion by felixkat** [templaza/astroid-framework#890] [Ideas]
It would be great if we could integrate the text element a little more with Joomla, i.e to use the standard Joomla editor and content plugins.  I'm not sure how feasible any of this would be but it's one of the missing links between using Joomla Articles and the article builder.

Long term, would it even be possible to link Joomla menu's directly to article layouts?  As far as I understand if you want to use the article builder for creating individual articles, you have to create a Joomla Article and Category for each one which doesn't seem logicial.

Just thinking out loud! 😄

**Comment by sonvnn** [comment]
Currently, because Astroid uses Vue Javascript Framework, loading Joomla Editor is quite a difficult problem. I am also researching it to find the best solution.

Identifying which Menu is displaying the article is not possible, but we can try with customizing Article Layout in each article to override Category data

**Reply by felixkat** [reply]
> Currently, because Astroid uses Vue Javascript Framework, loading Joomla Editor is quite a difficult problem. I am also researching it to find the best solution

It's mainly so I can add content plugins to the text widgets, now that I have that working.  For now as a workaround I create what I need in a Joomla article and then copy and paste it into a text widget which works fine.
